system-int-test/test/cmr/system_int_test/search/collection_with_new_granule_search_test.clj

```diff
@@ -0,0 +1,108 @@
+(ns cmr.system-int-test.search.collection-with-new-granule-search-test
+  "Integration test for searching collections created after a given date.
+   These tests are to ensure proper CMR Harvesting functionality."
+  (:require
+   [clj-http.client :as client]
+   [clojure.string :as string]
+   [clojure.test :refer :all]
+   [cmr.system-int-test.data2.core :as d]
+   [cmr.system-int-test.data2.granule :as dg]
+   [cmr.system-int-test.data2.umm-spec-collection :as data-umm-c]
+   [cmr.system-int-test.system :as s]
+   [cmr.system-int-test.utils.dev-system-util :as dev-system-util]
+   [cmr.system-int-test.utils.index-util :as index]
+   [cmr.system-int-test.utils.ingest-util :as ingest]
+   [cmr.system-int-test.utils.search-util :as search]
+   [cmr.system-int-test.utils.url-helper :as url-helper]))
+
+(use-fixtures :each (join-fixtures
+                      [(ingest/reset-fixture {"provguid1" "PROV1"})
+                       (dev-system-util/freeze-resume-time-fixture)]))
+
+(deftest search-for-collections-with-new-granules
+  (let [_ (dev-system-util/freeze-time! "2010-01-01T10:00:00Z")
+        oldest-collection (d/ingest-umm-spec-collection
+                            "PROV1"
+                            (data-umm-c/collection
+                              {:EntryTitle "oldie"
+                               :Version "v1"
+                               :ShortName "Oldie"}))
+
+        _ (dev-system-util/freeze-time! "2010-01-01T10:00:00Z")
```

Comment:
You don't need to call `freeze-time` before you save each one. Just call it before each group of concepts that you want to have the same timestamp.